app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -36,6 +36,20 @@ def create
 
       private
 
+      def validate_create_action
+        raise Exceptions::NotAuthorizedError, "Not authorized to create [#{params[:operation]}] action " unless valid_operation
```

Comment:
do we really need to split the logic into two methods?

Reply:
Changed into one method